spec.md

```diff
@@ -245,6 +243,32 @@ querying of the Service Broker's catalog, the Platform might not have an
 end-user with which to associate the request, therefore in those cases the
 originating identity header would not be included in those messages.
 
+## Service Broker Errors
+
+When a request to a Service Broker fails, the Service Broker MUST return an
+appropriate HTTP response code. If an appropriate response code is not defined
+within this specification, Service Broker authors SHOULD chose a code that
+accurately reflects the nature of the failure.
+
+For error responses, the following fields are valid:
```

Comment:
Does this prohibit broker specific error fields?

Reply:
Not at all, the broker may always add fields to the response, but it can't assume a platform will not ignore them.

Reply:
You are the native speaker. ;-)

But it wouldn't hurt to clarify this with a few sentences for people like me. Something along those lines: 
"Brokers MAY add more fields, which MUST be ignored by the platform.  Broker author SHOULD NOT use generic field names that may clash with future specification updates. It is recommended choose field names with a company prefix.”